packages/react/src/components/form/form-customisation.test.tsx

```diff
@@ -92,19 +95,21 @@ describe("#Form - customisation", () => {
 
     render(
       <TestSlashIDProvider sdkState="ready" logIn={logInMock}>
-        <Form>
-          <Slot name="error">
-            <Form.Error key="error slot child">
-              {({ context, retry, cancel }) => (
-                <div data-testid="custom-error-function">
-                  <h1>{context.error.message}</h1>
-                  <button onClick={() => retry("retry")}>Retry</button>
-                  <button onClick={cancel}>Cancel</button>
-                </div>
-              )}
-            </Form.Error>
-          </Slot>
-        </Form>
+        <TestTextProvider text={TEXT}>
```

Comment:
The interpolation tokens functionality depends on the `text` object in the context, otherwise it breaks